### ISSUE TITLE
Function to return list of bytes (i.e. List Int)

### ISSUE THREAD

**Issue by TSFoster** [truqu/elm-md5#2]
Hi,

Would you consider adding a second exposed function, something like `hexAsBytes` or `hexInOctets, which returns the digest as a `List Int`? For example:

```elm
MD5.hexAsBytes "hello world" ==
    [ 0x5e , 0xb6 , 0x3b , 0xbb
    , 0xe0 , 0x1e , 0xee , 0xd0
    , 0x93 , 0xcb , 0x22 , 0xbb
    , 0x8f , 0x5a , 0xcd , 0xc3
    ]
```

If so, I can make a PR if you want.

**Comment by TSFoster** [comment]
P.S. 59bdaa0a10d47f89dd15382d7e52666d502020bf is the commit that adds the function on my fork

**Comment by zwilias** [comment]
Hi @TSFoster!

Sorry for the slow response, I was on a holiday.

I added a `bytes` function in 93b72ee9b4ac1df382059a797fd8323d1fb22bc8 and published the addition in 1.1.0!

Thanks for the idea!

---

As a side-note, I tend to do this type of changes under close scrutiny with a lot of benchmarking in between to prevent performance from deteriorating while also trying to limit the number of bytes this package adds as a dependency. Hence my reluctance to request a PR - I hope you don't mind!

**Comment by TSFoster** [comment]
Thanks @zwilias, I hope others find it useful. Not a problem about no PR, makes sense!
